pkgs/development/tools/build-managers/xmake/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchgit
+# optional list of extra waf tools, e.g. `[ "doxygen" "pytest" ]`
+
+}:
+let
+in
```

```suggestion
}:
```

this empty let block doesn't do anything